nix-buffer.nix

```diff
@@ -0,0 +1,23 @@
+{ root }:
+let
+  defnix = root + "/default.nix";
+  pkgs = import <nixpkgs> {};
+  inherit (pkgs) lib writeText runCommand;
+  inherit (pkgs.emacsPackagesNg) inherit-local;
+  inherit (pkgs.nixBufferBuilders) withPackages;
+  drv = import (builtins.toPath defnix) {};
```

Comment:
@shlevy This will throw an error when default.nix doesn't have an arg (just starts with ```with import <nixpkgs> { };```). Any way to get Nix to pretend it's a function?